x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/components/step_panel/index.tsx

```diff
@@ -17,15 +17,19 @@ interface StepPanelProps {
 }
 
 const MyPanel = styled(EuiPanel)`
-  poistion: relative;
+  position: relative;
 `;
 
-export const StepPanel = memo<StepPanelProps>(({ children, loading, title }) => {
-  return (
-    <MyPanel>
-      {loading && <EuiProgress size="xs" color="accent" position="absolute" />}
-      <HeaderSection title={title} />
-      {children}
-    </MyPanel>
-  );
-});
+const StepPanelComponent: React.FC<StepPanelProps> = ({ children, loading, title }) => (
+  <MyPanel>
+    {loading && <EuiProgress size="xs" color="accent" position="absolute" />}
+    <HeaderSection title={title} />
+    {children}
+  </MyPanel>
+);
+
+StepPanelComponent.displayName = 'StepPanelComponent';
+
+export const StepPanel = memo(StepPanelComponent);
+
+StepPanel.displayName = 'StepPanel';
```

Comment:
Seems react-dev-tools will detect the memo and use the `displayName` of the component being wrapped (adding a `Memo` tag), so we don't need to add a display name for subsequent `memo`'s (unlike the behavior when wrapping components with `connect()`).

<img src="https://user-images.githubusercontent.com/2946766/71853482-d41c2d00-3098-11ea-9c54-bf2b343407d2.png" width=300 />